src/applications/caregivers/components/FormFields/AddressWithAutofill.jsx

```diff
@@ -93,20 +88,18 @@ const PrimaryAddressWithAutofill = props => {
     [addDirtyField],
   );
 
-  // check field for validation errors
+  // check for validation errors if field is dirty or form has been submitted
   const showError = field => {
-    const errorList = errorSchema[field].__errors;
```

Comment:
Removing this check to let fields with pattern validation run with case-insensitivity.